eberron.md

```diff
@@ -95,13 +104,14 @@ When cast at 9th level or above, your range spans the entire plane.
 #### Bloodied Blade
 *2nd level*
 
-- **Casting Time:** 5 AP, bladed weapon
+- **Casting Time:** 3 AP, bladed weapon
 - **Range:** self
-- **Duration:** instant
+- **Duration:** 1 minute
```

Comment:
I changed the duration/casting time as the spell also requires making an attack afterwards. That can be awkward having to do both actions, so I can change the spell text and leave the AP requirement at ~6..

while having them split is clunky, it lets people benefit from both Quickened Spellcasting and Fast Attacks, which you may not actually want with this.